docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,35 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 29 March 2023
+
+Frontend version: 1.0.0-20230322.1044 | Backend version: 1.55.6-230323.1031
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Tenants can retry or stop a webhook trigger on TechPass portal</summary>
+
+Tenants can now retry or stop a webhook trigger on TechPass portal. This option is useful when a code change required. Previously, the webhook was automatically re-triggered and expired after 14 days.
+
+Refer to [Retry or stop webhook trigger](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/webhooks?id=retry-or-stop-webhook) for more information.
+</details>
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> <i>user-deleted</i> webhook event now includes assigned groups</summary>
+
+The webhook event *user-deleted* includes all groups the user is assigned to that follow *namespace:group name*. This limits the groups to those belonging to a tenant and does not expose any TechPass internal groups the user is assigned to.
```

Comment:
Can we change it to:
The webhook event *user-deleted* includes all groups the user is assigned to that follow the *namespace:group name* naming convention. 

I don't think we need to announce about the internal groups.

Reply:
updated